lib/profile_node.stp

```diff
@@ -200,24 +200,29 @@ function get_shared_info_line_number(shared_info:long) {
  * Just grabbing the two-byte value and treating it as if it were ascii.
  * Works for most unicode chars!
  */
-function print_user_utf16_string(data_ptr:long, length:long) {
+function get_user_utf16_string(data_ptr:long, length:long) {
     data_ptr = strip_low_bit(data_ptr)
+    str = ""
     for (i = 0; i < 200 && i < length; i++) {
         c = user_uint16(data_ptr)
-        printf("%c", c);
+        str .= sprintf("%c", c);
```

Comment:
Not sure if it is a big deal and I'm not sure if I'm correct. But for better performance we might want to use a string builder type of thing here